cxx-qt-lib-headers/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+# SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+# SPDX-FileContributor: Be Wilson <be.wilson@kdab.com>
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0
+[package]
```

Comment:
Can you add more metadata here for when we publish, eg copy from cxx-qt-lib the license, repository, description, authors etc.

Reply:
no, that isn't applicable to virtual manifests; there is no `[package]` https://doc.rust-lang.org/cargo/reference/workspaces.html

Reply:
Hmm but it does have a `[package]` ? so `cxx-qt-lib-headers` is not a crate that would go to crates.io ?

Reply:
oh sorry I thought you were commenting on the workspace manifest

Reply:
added